docs/site/TEMP-TEXT-FILES/ddl.txt

```diff
@@ -0,0 +1,7 @@
+CLDR DDL Subcommittee
+The Common Locale Data Repository (CLDR) is widely used, and the content has grown dramatically over the years with participation by organizations of all types and sizes, as well as many individual contributors.
+Contributors for Digitally Disadvantaged Languages (DDL) face unique challenges. The CLDR-DDL subcommittee has been formed to evaluate mechanisms to make it easier for contributors for DDLs to:
+become contributors to CLDR
+improve the coverage for their language in CLDR
+raise the status of their contributions, so that the CLDR data for their language is incorporated into more products.
+The DDL Subcommittee has started to meet every other week as of June, 2023.
```

Comment:
`TEMP-TEXT-FILES` is not an appropriate directory. Please create a new PR deleting this directory, also with the correct commit message.